tests/Factory/ModelsFactory.php

```diff
@@ -256,6 +258,36 @@ public static function createQuestions(): Questions
         );
     }
 
+    /**
+     * @return Registration
+     * @throws \Exception
+     */
+    public static function createRegistration(): Registration
+    {
+        return new Registration(
+            Uuid::fromString('00f20af9-c2f5-4bfb-9424-5c0c29fbc2e3'),
+            'd2c63a605ae27c13e43e26fe2c97a36c4556846dd3ef',
+            self::createUser(),
+            new \DateTimeImmutable('2020-02-02'),
+            false
+        );
+    }
+
+    /**
+     * @return Registration
+     * @throws \Exception
+     */
+    public static function createRegistrationWithAlternateUser(): Registration
+    {
+        return new Registration(
+            Uuid::fromString('00f20af9-c2f5-4bfb-9424-5c0c29fbc2e3'),
+            'd2c63a605ae27c13e43e26fe2c97a36c4556846dd3ef',
+            self::createAlternateUser(),
+            new \DateTimeImmutable('2020-02-02'),
```

Comment:
@stejes best een echte date time aanmaken inclusief tijdszone om zeker te zijn dat de tijdszone goed wordt bewaard in SQL.